src/lib.rs

```diff
@@ -86,12 +86,10 @@ pub fn get_timezone() -> std::result::Result<String, crate::GetTimezoneError> {
     platform::get_timezone_inner()
 }
 
-#[cfg(test)]
-mod tests {
```

Comment:
why did we unwrap this outside of the cfg'd `mod tests`?

Reply:
My thinking was, that someone might add tests to this `mod` that don't get executed because of the `not(android)` guard, even though the tests might be useful on android, too. I.e. that the `not(android)` part was too easy to overlook. I guess that's overthinking it.

Probably it would have been even smarter to add the guard to the actual test and let the `mod` stay. I'll fix that. :)